assets/js/translator.js

```diff
@@ -554,10 +554,19 @@ function populateTranslationList() {
 
     var minColumnWidth = TRANSLATION_LIST_MAX_WIDTH / TRANSLATION_LIST_MAX_COLUMNS;
 
+    var maxSrcLangsWidth, maxDstLangsWidth;
+
     // figure out how much space is actually available for the columns
-    var maxSrcLangsWidth = $(window).width() - $('#srcLanguagesDropdownTrigger').offset().left - TRANSLATION_LISTS_BUFFER,
+    if(langDirection(locale) === 'ltr') {
```

Comment:
This issue is that Flow knows that `locale` might not be initialized by the time this function gets called.

Reply:
Makes sense. Though how would I fix that?

Reply:
Does this code only get called after the `locale` gets initialized? If so, we should just throw here if it's `undefined` and Flow will use that as signal for its analysis.

Reply:
I just tested and it gets called once before `locale` gets initialized and once after. But I I have thought of a way to fix this

Reply:
Okay done